xrpl/caip122.md

```diff
@@ -30,6 +30,16 @@ Correspondingly, a message needs to be serialized to bytes prior to signing it w
 
 We propose using the signature type `xrpl:secp256k1` and `xrpl:ed25519` to refer to the chain and algorithm used uniquely.
```

Comment:
I would maybe add a sentence to this paragraph like, 
```
These two signature types are the only two available and can be considered the defaults per key type. The typing of the signature MAY be made explicit by including a `SignatureMeta.t` property set to either `xrp1:secp256k1` or `xrp1:ed25519`, but if omitted the appropriate one of these two defaults should be obvious from the keytype provided in the `SignatureMeta.SigningPubKey` value.
```

Reply:
I think the context of your sentence is covered in the `Signing Algorithm` section.